src/plugins/wip/index.js

```diff
@@ -0,0 +1,181 @@
+/**
+ * @fileoverview Handle PR status for works in progress, using either "WIP" in title or "do not merge" label
+ * @author Kevin Partington
+ */
+
+"use strict";
+
+const WIP_IN_TITLE_REGEX = /^WIP:|\(WIP\)/i;
+const DO_NOT_MERGE_LABEL = "do not merge";
+
+/**
+ * Create status on the PR
+ * @param {Object} context - probot context object
+ * @param {string} state - state can be either success or failure
+ * @param {string} sha - sha for the commit
+ * @param {string} description - description for the status
+ * @param {string} targetUrl The URL that the status should link to
+ * @returns {Promise} Resolves when the status is created on the PR
+ * @private
+ */
+function createStatusOnPR({ context, state, sha, description, targetUrl }) {
+    return context.github.repos.createStatus(
+        context.repo({
+            sha,
+            state,
+            target_url: targetUrl || "",
```

Comment:
Is a target URL ever provided here? I'm not sure of the effect of using an empty string as the target URL (I would hope that it doesn't display any details, but it's possible that it creates something like `a href=""` which would just link to the current page).